packages/editor-ui/src/composables/useNodeHelpers.ts

```diff
@@ -97,11 +97,13 @@ export function useNodeHelpers() {
 
 		if (!isObject(parameters)) return false;
 
-		if ('resource' in parameters && 'operation' in parameters) {
+		if ('resource' in parameters || 'operation' in parameters) {
```

Comment:
since `isCustomApiCallSelected` is exported, and since this composable has a test file already, can we please add a test for `isCustomApiCallSelected` to prevent this from happening again 🙏🏽

Reply:
Maybe @elsmr can pitch in :)
Else I'll come to this later

Reply:
He's OOO